frontend/src/features/comparisons/comparisonSlice.ts

```diff
@@ -0,0 +1,36 @@
+import { RootState } from '../../app/store';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { PollStats } from 'src/utils/types';
+import { DEFAULT_POLL_STATS } from 'src/utils/api/stats';
+
+export const initialState: PollStats = DEFAULT_POLL_STATS;
+
+export const comparisonSlice = createSlice({
```

Comment:
This slice looks more like a `statsSlice` rather than a `comparisonSlice`.

I'd suggest to create a `statsSlice` instead, as the public statistics of the Tournesol API can contain different kind of data, like entities and users, and not only comparisons.